chapters/1.md

```diff
@@ -1,100 +1,185 @@
 
-To look at the history of Monero, we have to briefly start with a cryptocurrency named Bytecoin based on the CryptoNote protocol. Like Bitcoin, the creator of Bytecoin is anonymous. It is believed that Bytecoin was created around the time a Bitcointalk thread was established to introduce the cryptocurrency in 2014.
+Right now, privacy is conspicuously absent from mainstream economic and commercial systems. Traditional payment processors, banks, and cryptocurrencies leave very clear trails that are used to study, surveil, and profit from you. Once collected, you often have no way to control or track the proliferation of your data, or the privacy and personal security risks that arise from its sale to unknown parties.
 
-Soon after its introduction, members of the blockchain and cryptocurrency space began viewing Bytecoin as a project and cryptocurrency that lacked credibility, due to having been pre-mined. A member of Bitcointalk, “thankful_for_today”, did some calculations about emission and noted that Bytecoin had emitted approximately 82 percent of its coins. With a pre-mine of this size, potential centralization of the coins in circulation becomes a risk.
+The only guaranteed way to exercise your right to financial privacy is to avoid revealing personal information in the first place! To stay safe, we need a way to interact securely - where transactions cannot be linked to your identity, your wealth, or other transactions. The Monero cryptocurrency is your best tool for taking all of these matters into your own hands!
 
-In light of this, “thankful_for_today” decided to fork Bytecoin in order to launch a cryptocurrency with a strong, community-driven development team working off of the CryptoNote protocol. On April 18th, 2014, “thankful_for_today” launched BitMonero and on April 30th, 2014, the name of the cryptocurrency was changed to Monero.
+## Introducing Monero
 
-During January 2017, the privacy of transactions was strengthened further with the optional use of the Ring Confidential Transactions (RingCT) algorithm, which introduced an additional layer of confidentiality by not displaying the amounts implicated in a transaction to someone who did not directly take part in it. By early February 2017, over 95% of all non-coinbase transactions used the optional RingCT feature. As of today, RingCT is a default feature and is no longer optional.
+[infographic]
 
- ## Why Monero?
+Monero (pronounced /mōnĕrō/, plural “Moneroj”) is a leading cryptocurrency with a focus on private and censorship-resistant transactions. The openly verifiable nature of most cryptocurrencies (such as Bitcoin and Ethereum) allows anybody in the world to track your money. Furthermore, links between your financial records and personal identity may jeopardize your safety.
```

Comment:
Possibly add the pronunciation of "Moneroj" since it's not evident. (Most people don't know how to pronounce Esperanto correctly.)